Sources/StreamChat/Controllers/ListDatabaseObserver.swift

```diff
@@ -285,7 +284,12 @@ class ListChangeAggregator<DTO: NSManagedObject, Item>: NSObject, NSFetchedResul
         for type: NSFetchedResultsChangeType,
         newIndexPath: IndexPath?
     ) {
-        guard let dto = anObject as? DTO, let item = itemCreator(dto) else {
+        guard let dto = anObject as? DTO else {
+            log.warning("Skipping the update from DB because anObject can't be converted to DTO.")
+            return
+        }
+
+        guard let item = itemCreator(dto) else {
             log.warning("Skipping the update from DB because the DTO can't be converted to the model object.")
             return
         }
```

Comment:
We don't want this. We want the `item` to exist, if a relationship is not found for example in `ChannelDTO` it doesn't mean the channel can't be created, it only mean it will have some properties `nil`